pyclam/manifold.py

```diff
@@ -569,6 +567,96 @@ def clear_cache(self) -> None:
         self.cache.clear()
         return
 
+    def as_dot_string(
```

Comment:
I don't know what I'm looking for here, this function looks OK, but a large part of that is just trusting you